assembl/views/discussion/views.py

```diff
@@ -182,16 +182,71 @@ def react_view(request):
     Basic view for the homepage
     """
     old_context = base_default_context(request)
-    
+    user_id = authenticated_userid(request) or Everyone
```

Comment:
Same comment: That's best done in a single commit, outside this PR. Good catch otherwise!